playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -14,5 +14,5 @@ elasticsearch_group: "elasticsearch"
 # Defaults for a single server installation.
 ELASTICSEARCH_CLUSTER_MEMBERS: []
 ELASTICSEARCH_CLUSTER_NAME: 'localhost'
-ELASTICSEARCH_HEAP_SIZE: "512m"
+ELASTICSEARCH_HEAP_SIZE: "8g"
```

Comment:
Actually, this would effect devstacks too (they don't have that much RAM).

We should instead do this in the relevant `group_vars` dirs...